### PR TITLE
428 project fixes

### LINKED ISSUE #405: Improve organisation dedup RSD/ROR
When searching for an organisation the same organisation could be shown twice using RSD or ROR as source. The current key is slug based on organisation name and it causes some errors in the console

image

### LINKED ISSUE #421: Sizing issue when using certain SVGs as organisation logos
In the organsations grid, sometimes there are scaling issues when using certain SVG files, e.g. this one:

DLR_short_en

Here it is in action. The GFZ logo is an SVG as well but has no scaling issues:

image

/cc @dmijatovic

### LINKED ISSUE #422: Use singular when last commit is 1 [unit] away
The default setting for displaying the information below the commit graph is to use the plural of the unit: years, months, days:

image

If the number in front is 1, we should use the singular.

### LINKED ISSUE #428: Edit project - information section bugs (feedback Rena)
# Information section 
1. Cannot add funding organisation to project. When adding funding organisation, the item is shown in the edit page as added. But it does not appear in the project view at the right panel. After reopening project edit page, the funding organisation is not shown anymore as added. It seems it is never been saved.
2. user cannot add keywords on the project page. Similar issue as with funding organisations. 
3. user cannot update label of created link. After using save button new label value is not saved.

### PR DESCRIPTION
**Pull request by dmijatovic**
# Fixes based on feedback

Closes #405
Closes #421  
Closes #422 
Closes #428

Changes proposed in this pull request:
* See  mentioned issues     

How to test:
* `docker-compose build` to build
* `docker-compose up` to start
* confirm you can add keywords, funding organisations and add/update project urls

# Example #421 logo fix DLR
image

PR Checklist:

*   [x] Increase version numbers in `docker-compose.yml`
*   [x] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

### CODE REVIEW

**Comment by dmijatovic** [comment]
@cmeessen Can you have a look at this PR? I would like to merge it before start working on #418

**Comment by cmeessen** [comment]
> @cmeessen Can you have a look at this PR? I would like to merge it before start working on 418

Yes, I am on it but am frequently interrupted :)

**Review by cmeessen — APPROVED** [review]
Looking good. I coudln't test #422 because my database was empty and there was no repo scraped with such a condition. On the code level it looks good though.

File: deployment/.gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@
 .env.*
 !.env.example
 
+hmz
+nle
```
Comment (cmeessen) [inline deployment/.gitignore:14]: Out of curiosity: what are `hzm` and `nle`?

Comment (dmijatovic) [reply deployment/.gitignore:14]: These are folders I used locally to mount Helmholtz and eScience themes and fonts. I do not want these folders to be commited to repo. I send you an email with the content of hmz folder so you can test.

**Pull request merged**